packages/@aws-cdk/aws-lambda/lib/log-retention.ts

```diff
@@ -26,6 +26,31 @@ export interface LogRetentionProps {
    * @default - A new role is created
    */
   readonly role?: iam.IRole;
+
+  /**
+   * Retry options for managing CloudWatch log groups.
+   *
+   * @default - AWS SDK default retry options
+   */
+  readonly logRetentionRetryOptions?: LogRetentionRetryOptions;
+}
+
+/**
+ * Retry options for managing CloudWatch log groups
+ */
+export interface LogRetentionRetryOptions {
+  /**
+   * The maximum amount of retries.
+   *
+   * @default - AWS SDK default
+   */
+  readonly maxRetries?: number;
```

Comment:
Just to clarify - if `maxRetries` is 5, there will be 6 attempts, yes?

Reply:
Correct